tests/interface/__init__.py

```diff
@@ -0,0 +1,3 @@
+# ~ import pytest
+
+# ~ pytestmark = pytest.mark.skip("Skipping for hangs")

```

Comment:
Should we delete all these commented out skips or do you still need them for more testing?

Reply:
Those can definitely go -- I'll raise a small PR. (Sorry; I've gone backwards and forwards so much over the code, trying to narrow down the test failures that I forgot about them..)